fmriprep/workflows/bold/base.py

```diff
@@ -935,6 +1009,8 @@ def init_func_derivatives_wf(output_dir, output_spaces, template, freesurfer,
 
 def _get_series_len(bold_fname):
     from niworkflows.interfaces.registration import _get_vols_to_discard
+    if isinstance(bold_fname, list):  # Multi-echo data
+        bold_fname = bold_fname[0]
```

Comment:
`sorted(bold_fname)[0]` perhaps? is it relevant the order at all here?

Reply:
The order wouldn't be relevant here, as all echo time series should have the same number of volumes !